src/util.ts

```diff
@@ -0,0 +1,104 @@
+import { Keyring, WsProvider } from "@polkadot/api";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+import assert from "assert";
+
+import { OPENGOV_LARGE_TIP_VALUE, OPENGOV_MEDIUM_TIP_VALUE, OPENGOV_SMALL_TIP_VALUE } from "./constants";
+import { ContributorAccount, OpenGovTrack, State, TipMetadata, TipNetwork, TipRequest, TipSize } from "./types";
+
+const validTipSizes: { [key: string]: TipSize } = { small: "small", medium: "medium", large: "large" } as const;
+
+export function getTipSize(tipSizeInput: string | undefined): TipSize {
+  if (!tipSizeInput || !(tipSizeInput in validTipSizes)) {
+    throw new Error(`Invalid tip size. Please specify one of ${Object.keys(validTipSizes).join(", ")}.`);
+  }
+
+  return validTipSizes[tipSizeInput];
+}
+
+export function tipSizeToOpenGovTrack(tipSize: TipSize): { track: OpenGovTrack; value: number } {
+  if (tipSize === "small") return { track: "SmallTipper", value: OPENGOV_SMALL_TIP_VALUE };
+  if (tipSize === "medium") return { track: "BigTipper", value: OPENGOV_MEDIUM_TIP_VALUE };
+  if (tipSize === "large") return { track: "BigTipper", value: OPENGOV_LARGE_TIP_VALUE };
```

Comment:
Hmm I'm confused about this, I thought that the `small tipper` track goes up to 8.25 of [the native currency of the chain we look at].

So 8.25 DOT in Polkadot and 8.25 KSM on Kusama.
Is that not right?

Reply:
no, depends on the chain config. 
8.333 is the result of this line https://github.com/paritytech/polkadot/blob/e164da65873f11bf8c583e81f6d82c21b005cfe4/runtime/kusama/src/governance/origins.rs#L172

in polkadot it will be https://github.com/paritytech/polkadot/blob/e164da65873f11bf8c583e81f6d82c21b005cfe4/runtime/polkadot/src/governance/origins.rs#L143
where DOLLARS are https://github.com/paritytech/polkadot/blob/e164da65873f11bf8c583e81f6d82c21b005cfe4/runtime/polkadot/constants/src/lib.rs#L31

`10_000_000_000e-10 * 250 = 250 DOT`

which tells me that for polkadot just a small tipper will be more than enough